### PR TITLE
chore(demo-cypress): screenshot testing for component tests

### PR DESCRIPTION
**Pull request by nsbarsukov**

### CODE REVIEW

File: projects/demo/src/modules/components/chip/examples/4/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     class="fade"
 >
     <div tuiFade>Very long value in chip</div>
-    <div>{{ 123435 | tuiAmount: 'RUB' | async }}</div>
+    <div>{{ 123456 | tuiAmount: 'RUB' | async }}</div>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/chip/examples/4/index.html:6]: Just to debug that Playwright screenshot testing still properly works in CI

File: projects/demo/package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
         "@stackblitz/sdk": "1.11.0",
         "@taiga-ui/dompurify": "4.1.7",
         "date-fns": "4.1.0",
-        "rxjs": "7.5.0"
+        "rxjs": "7.8.1"
```
Comment (nsbarsukov) [inline projects/demo/package.json:21]: `7.5.0` is not compatible with `"moduleResolution": "bundler"`

**Review by splincode — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -156,7 +156,7 @@
         "ng-packagr": "16.2.3",
         "ngx-highlightjs": "10.0.0",
         "nx": "19.8.2",
-        "rxjs": "7.5.0",
+        "rxjs": "7.8.1",
```
Comment (nsbarsukov) [inline package.json:159]: `7.5.0` is not compatible with `"moduleResolution": "bundler"`

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/chip/examples/4/index.html (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/components/chip/examples/4/index.html:6]: Still works 👍 
Снимок экрана 2024-10-01 в 16 14 27

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**